content_script.js

```diff
@@ -28,6 +40,74 @@ if (document.getElementById('ROLL_TOGETHER_SCRIPT') == null) {
       return { state, currentProgress, timeJump };
     }
 
+   const getSkipButtonState = (currentProgress) => {
+      if(!beginIntro) return skipButtonStates.HIDDEN;
```

Comment:
It's ok to leave like this, most of the code on this base does that but usually, it's better to have a function that says if something is null because other things like 0 (that could be valid) and "" will also go inside this if.
In this case you can do beginIntro === null but overall it's better to use a function like this https://lodash.com/docs/4.17.15#isNull

Reply:
Ok, changed